projects/chaos/src/sequencer_fsm.c

```diff
@@ -105,14 +152,21 @@ static const SequencerEventPair s_charge_events[] = {
 };
 
 // Drive State Order:
+// - Turn off the charger
+// - Open all GPIO that shouldn't be on.
 // - Close battery relay (DCDC)
 // - Enable DCDC monitoring
 // - Turn on drive GPIOs
 // - Close solar master rear relay
 // - Close solar master front relay
-// - Close main power relay
-// NOTE: No need to manipulate the charger since you can't reach Drive from Charge.
+// - Close motor relay
 static const SequencerEventPair s_drive_events[] = {
+  { .raise = { .id = CHAOS_EVENT_SEQUENCE_DRIVE_START, .data = SEQUENCER_EMPTY_DATA },
+    .response = SEQUENCER_NO_RESPONSE },
+  { .raise = { .id = CHAOS_EVENT_CHARGER_OPEN, .data = SEQUENCER_EMPTY_DATA },
+    .response = SEQUENCER_NO_RESPONSE },
+  { .raise = { .id = CHAOS_EVENT_GPIO_DRIVE_PRECONFIG, .data = SEQUENCER_EMPTY_DATA },
```

Comment:
Do you need to add a handler for these?

Reply:
It'll be in a separate GPIO FSM refactoring.